descriptions.go

```diff
@@ -126,11 +131,255 @@ type RequestorInfo struct {
 	Hostnames  []string                  `json:"hostnames"`
 	Logo       *string                   `json:"logo"`
 	ValidUntil *Timestamp                `json:"valid_until"`
+	Wizards    map[string]*IssueWizard   `json:"wizards"`
 }
 
 // RequestorChunk is a number of verified requestors stored together. The RequestorScheme can consist of multiple such chunks
 type RequestorChunk []*RequestorInfo
 
+type (
+	IssueWizard struct {
+		ID       string                    `json:"id"`
+		Title    TranslatedString          `json:"title"`
+		Logo     *string                   `json:"logo,omitempty"`
+		LogoPath *string                   `json:"logoPath,omitempty"`
+		Issues   *CredentialTypeIdentifier `json:"issues,omitempty"`
+
+		Info *TranslatedString `json:"info,omitempty"`
+		FAQ  []IssueWizardQA   `json:"faq,omitempty"`
+
+		Intro              *TranslatedString   `json:"intro,omitempty"`
+		SuccessHeader      *TranslatedString   `json:"successHeader,omitempty"`
+		SuccessText        *TranslatedString   `json:"successText,omitempty"`
+		ExpandDependencies *bool               `json:"expandDependencies,omitempty"`
+		Contents           IssueWizardContents `json:"contents"`
+	}
+
+	IssueWizardQA struct {
+		Question TranslatedString `json:"question"`
+		Answer   TranslatedString `json:"answer"`
+	}
+
+	IssueWizardContents [][][]IssueWizardItem
+
+	IssueWizardItem struct {
+		Type       IssueWizardItemType       `json:"type"`
+		Credential *CredentialTypeIdentifier `json:"credential,omitempty"`
+		Header     *TranslatedString         `json:"header,omitempty"`
+		Text       *TranslatedString         `json:"text,omitempty"`
+		Label      *TranslatedString         `json:"label,omitempty"`
+		SessionURL *string                   `json:"sessionUrl,omitempty"`
+		URL        *TranslatedString         `json:"url,omitempty"`
+		InApp      *bool                     `json:"inapp,omitempty"`
+	}
```

Comment:
Checking for the presence of irrelevant options is a good idea, but the `Credential` field is never irrelevant: it may be present also on items of type `website` and `session` (of which the latter will contain a `sessionURL`). In each of those cases, it means that the item is completed when the mentioned credential type has been issued. `irmamobile` can use this to mark an issuance wizard item as completed in the wizard screen.

Reply:
Ah okay, and if the `Credential` option is not supplied, how will `irmamobile` then conclude that the task is finished? SImply on return in the IRMA-app?

Reply:
[This comment](https://github.com/privacybydesign/irmago/pull/118#discussion_r563881964) answers this.